prototypes/water_rails.lua

```diff
@@ -136,17 +136,16 @@ data:extend({
     name = "straight-water-way",
     icon = GRAPHICSPATH .. "icons/water_rail.png",
     icon_size = 64,
-    flags = {"placeable-neutral", "player-creation", "building-direction-8-way", "not-on-map"},
+    flags = {"placeable-neutral", "player-creation", "building-direction-8-way"},
     resistances = invincible,
-    minable = {mining_time = 0.2, result = "water-way"},
+    minable = {mining_time = 0.1},
```

Comment:
Only reason for a longer mining time is to prevent accidental clicks from mining a hole in your rail network.  If you need to do a lot of them the deconstruction planner will be faster.